static/src/jspm-config.js

```diff
@@ -0,0 +1,135 @@
+System.config({
+  "baseURL": "/assets",
+  "transpiler": "babel",
+  "paths": {
+    "*": "javascripts/projects/*.js",
+    "es6/*": "javascripts/es6/*.js",
+    "bootstraps/*": "javascripts/bootstraps/*.js",
```

Comment:
Was this edited by hand here? Is it safe to do so? (It used not to be, but jspm may have got smarter)